app/views/layouts/application.html.erb

```diff
@@ -12,8 +12,8 @@
   <% flash.each do |type, msg| %>
     <%= content_tag :div, msg, class: type %>
   <% end %>
-  <p>Cart: <%= @cart.total_count %></p>
-
+  <p> <%= @cart.total_count %></p>
+  <%= button_to "View Cart:", cart_path(cart: @cart.contents) %>
```

Comment:
<%= button_to "View Cart:", cart_path(session[:cart), method: :get %>
solved our problem. @carmer @s-espinosa
now trying to make all the items in cart show up in view....